ios/test_app.rb

```diff
@@ -136,8 +137,10 @@ def generate_info_plist!(project_root, target_platform, destination)
   font_files = ['.otf', '.ttf']
   fonts = []
   resources = resolve_resources(manifest, target_platform)
-  resources.each do |filename|
-    fonts << File.basename(filename) if font_files.include?(File.extname(filename))
+  unless resources.nil?
```

Comment:
This was needed because if you don't specify `visionos` in your `app.json`, it errors out evaluating `resources.each`

Reply:
https://github.com/microsoft/react-native-test-app/pull/1847